frameworks/digital-outcomes-and-specialists-3/questions/declaration/serviceStandard.yml

```diff
@@ -1,7 +1,7 @@
 name: Digital Service Standard
 question: >
-  Are you aware of, and will you comply with, the [Digital Service Standard](https://www.gov.uk/service-manual/service-standard) 
-  and the government’s latest [service design principles](https://www.gov.uk/design-principles)?
+  Are you aware of, and will you comply with, the <a href="https://www.gov.uk/service-manual/service-standard" target="_blank" rel="noopener noreferrer">Digital Service Standard (link opens in a new tab)</a>
+  and the government’s latest <a href="https://www.gov.uk/design-principles" target="_blank" rel="noopener noreferrer">service design principles (link opens in a new tab)</a>?
```

Comment:
I'm sure there's some way we can kick markdown to just do this for links by default...

Reply:
I checked. It can't. Silly markdown.